src/target/adiv5.h

```diff
@@ -177,6 +177,12 @@
 /* AP Debug Base Address Register (BASE) */
 #define ADIV5_AP_BASE_BASEADDR UINT32_C(0xfffff000)
 #define ADIV5_AP_BASE_PRESENT  (1U << 0U)
+#define ADIV5_AP_BASE_FORMAT   (1U << 1U)
+
+#define ADIV5_AP_BASE_PRESENT_NO_ENTRY (0U << 0U)
+#define ADIV5_AP_BASE_FORMAT_LEGACY    (0U << 1U)
+#define ADIV5_AP_BASE_FORMAT_ADIv5     (1U << 1U)
```

Comment:
We've been pondering about that.. the front part makes sense because consistency with the other defines, but the latter part feels a bit odd honestly - and we have precedent for lower-case in a define with things like `LPC43x0`.. If you think it would be better to be brutally consistent though, we'll get this changed. Same for the remote protocol one.

Reply:
The lower case also breaks highlighter making things stand out and indicate an issue where there is actually not a problem, tripping me up during review. I understand your concern and I would agree if it was not tripping me up. :D So I would rather be brutally consistent, and also do the same fix for the number placeholders too.

Reply:
The clang-tidy does not like the inconsistent case either, so being more consistent reduces how many NOLINTs are required